Realm/Realm.Sync/Realm.Sync.csproj

```diff
@@ -6,7 +6,7 @@
     <ProjectGuid>{2015403A-59D2-47F2-8E18-7DF85BB98467}</ProjectGuid>
     <ProjectTypeGuids>{786C830F-07A1-408B-BD7F-6EE04809D6DB};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
     <OutputType>Library</OutputType>
-    <RootNamespace>Realm.Sync</RootNamespace>
+    <RootNamespace>Realms.Sync</RootNamespace>
```

Comment:
The `Realms.Sync` namespace change needs pulling into a separate and sooner PR of its own!